data/net/minecraft/world/level/storage/loot/GsonAdapterFactory.mapping

```diff
@@ -35,6 +35,11 @@ CLASS net/minecraft/world/level/storage/loot/GsonAdapterFactory
 			ARG 4 typeGetter
 			ARG 5 defaultType
 			ARG 6 inlineType
+		METHOD deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
+			ARG 1 json
+			ARG 3 deserializationContext
```

Comment:
```suggestion
			ARG 1 json
			ARG 2 typeOfT
			ARG 3 context
```
These type names are inherited from gson's [`JsonDeserializer`](https://www.javadoc.io/doc/com.google.code.gson/gson/2.6.2/com/google/gson/JsonDeserializer.html#deserialize-com.google.gson.JsonElement-java.lang.reflect.Type-com.google.gson.JsonDeserializationContext-)

Reply:
Oh, I didn't even make the connection.